<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="MatchFrame" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>760</width>
     <height>150</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>150</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>760</width>
     <height>150</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#MatchFrame{
border: 3px solid rgb(116, 128, 139);
border-radius: 10px;
background-color: rgb(90, 99, 108);
}

</string>
   </property>
   <widget class="QLabel" name="CharIMG">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Champ/Yuumi.webp&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="Result">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;KoPub돋움체 Bold&quot;;
color: rgb(255, 255, 255);
background-color: rgb(90, 99, 108);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;1위&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="MatchTime">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>116</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;KoPub돋움체 Bold&quot;;
color: rgb(255, 255, 255);
background-color: rgb(90, 99, 108);</string>
    </property>
    <property name="text">
     <string>99분 99초</string>
    </property>
   </widget>
   <widget class="QLabel" name="KDA">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;KoPub돋움체 Bold&quot;;
color: rgb(255, 255, 255);
background-color: rgb(90, 99, 108);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;99 / &lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;99&lt;/span&gt;&lt;span style=&quot; font-weight:600;&quot;&gt; / 99&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="KDALate">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>30</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;KoPub돋움체 Bold&quot;;
color: rgb(255, 255, 255);
background-color: rgb(90, 99, 108);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;KDA: 999,9&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="DMG">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>60</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;KoPub돋움체 Bold&quot;;
color: rgb(255, 255, 255);
background-color: rgb(90, 99, 108);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;DMG: 99999&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="MMR">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>110</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;KoPub돋움체 Bold&quot;;
color: rgb(255, 255, 255);
background-color: rgb(90, 99, 108);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;MMR: 9999 &lt;/span&gt;&lt;span style=&quot; font-size:12pt; color:#ff0000;&quot;&gt;▲999&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="TK">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;KoPub돋움체 Bold&quot;;
color: rgb(255, 255, 255);
background-color: rgb(90, 99, 108);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;TK: 99&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="KDAShow">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;KoPub돋움체 Bold&quot;;
color: rgb(255, 255, 255);
background-color: rgb(90, 99, 108);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#c3c3c3;&quot;&gt;K / D / A&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../UIResource/champIcons.qrc"/>
 </resources>
 <connections/>
</ui>
